packages/render/src/browser/render.tsx

```diff
@@ -1,78 +1,37 @@
 import { convert } from "html-to-text";
-import type {
-  PipeableStream,
-  ReactDOMServerReadableStream,
-} from "react-dom/server";
 import { Suspense } from "react";
-import { pretty } from "../shared/utils/pretty";
+import { renderToReadableStream } from "react-dom/server";
```

Comment:
Saw you removed the dynamic import back into this, are we sure that this isn't going to bring #871 back?